nn/conformer.py

```diff
@@ -14,21 +14,20 @@ class _PositionwiseFeedForward(nn.Module):
       FF -> Activation -> Dropout -> FF
   """
 
-  def __init__(self, dim_model: int, dim_ff: int, dropout: float, activation, l2: float = 0.0):
+  def __init__(self, dim_model: int, dim_ff: int, dropout: float, activation):
     """
     :param dim_model:
     :param dim_ff:
     :param dropout:
     :param activation:
-    :param l2:
     """
     super().__init__()
 
     self.dropout = dropout
     self.activation = activation
 
-    self.linear1 = nn.Linear(n_out=dim_ff, l2=l2)
-    self.linear2 = nn.Linear(n_out=dim_model, l2=l2)
+    self.linear1 = nn.Linear(n_out=dim_ff)
+    self.linear2 = nn.Linear(n_out=dim_model)
```

Comment:
See my comment on Transformer on this. I think there could be better names.
```suggestion
    self.linear_ff = nn.Linear(n_out=dim_ff)
    self.linear_out = nn.Linear(n_out=dim_model)
```